app/assets/javascripts/teams.js

```diff
@@ -50,7 +51,7 @@ jQuery(function ($) {
     });
   });
 
-  $('body').on('click', '.btn-edit-webhook', function (_event) {
+  $('body').on('click', '.btn-edit-webhook', function (event) {
     if ($(this).hasClass('button_edit_webhook')) {
       $('.webhook_information').toggle();
       $('#update_webhook_' + event.currentTarget.value).toggle();
```

Comment:
Yeah, good catch 👍  Note that we write variables starting with _ when we have to have a function parameter but we don't use it (see my comment above). And it looks like I didn't see this usage.